init.tf

```diff
@@ -2,8 +2,8 @@ terraform {
   required_providers {
     azurerm = {
       source                = "hashicorp/azurerm"
-      version               = ">= 3.7.0"
-      configuration_aliases = [azurerm.hub, azurerm.ssptl, azurerm.cftptl, azurerm.soc, azurerm.cnp]
+      version               = ">= 3.116.0"
```

Comment:
Should we replace `>=` with `=` in the version constraint?

Reply:
so since this is module, any provider version constraint will get overridden by the consumer so incase the consumer  wants to use higher version, it can do that.  If we keep equal to or less than sign, the consumers would get no choice

Reply:
Nice one, good to know!